ui/statistics/src/main/java/de/danoeh/antennapod/ui/statistics/feed/FeedStatisticsDialogFragment.java

```diff
@@ -26,6 +29,10 @@ public static FeedStatisticsDialogFragment newInstance(long feedId, String feedT
     public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
         AlertDialog.Builder dialog = new AlertDialog.Builder(getContext());
         dialog.setPositiveButton(android.R.string.ok, null);
+        dialog.setNegativeButton(R.string.open_podcast, (dialogInterface, i) -> {
```

Comment:
Opening the feed is not really a negative action. Neutral could be a closer (but still not perfect) match.

Reply:
Well the API naming doesn't really help, but I used the neutral as suggested and at least it's really aligned to the left now.